docs/LogGuide.md

```diff
@@ -1376,6 +1522,21 @@ ID | Name | Sample Locations | Consistent meaning?
 00BD | Purple Spread Circle (giant) | TItania N/EX | Yes
 00BF | Granite Gaol | e4s | N/A
 
+#### Offset Headmarkers
+
+Newer content uses 'sequential headmarkers' - every headmarker ID is offset by a per-instance value.
+You will need to wait until you see the first headmarker in the instance,
+and then use this as an offset by which to adjust all the other IDs you see.
+There are a few strategies for dealing with this in triggers and trigger platforms:
+
+- Figure out the real ID for the first headmarker you'd see in the instance,
+  and use this to calculate the real ID for all other markers in the instance.
+- Capture the ID of the first headmarker,
+  and subtract this from all subsequent headmarkers,
+  resulting in everything using relative values.
+- Most mechanics apply their markers in a consistent order,
+  so the order of the headmarkers can be used in lieu of the IDs.
+
```

Comment:
It's probably worth adding a note here that offset/sequential headmarkers are applied to newer content on release, and then removed later.

Reply:
Added